Makefile.validation

```diff
@@ -37,6 +37,9 @@ test-ci:
 test-short-ci:
 	${GO_ENV_VARS} CI=true CSI_VERSION=${CSI_VERSION} OPERATOR_VERSION=${OPERATOR_VERSION} go test -v test/e2e/baremetal_e2e_test.go -ginkgo.v -ginkgo.progress -ginkgo.failFast -timeout-short-ci=${SHORT_CI_TIMEOUT} -kubeconfig=${HOME}/.kube/config -chartsDir ${CHARTS_DIR} -timeout 0 > log.txt
 
+test-sanity-ci:
+	${GO_ENV_VARS} CI=true CSI_VERSION=${CSI_VERSION} OPERATOR_VERSION=${OPERATOR_VERSION} go test -v test/e2e/baremetal_e2e_test.go -ginkgo.focus=${SANITY_TEST} -ginkgo.v -ginkgo.progress -all-tests  -kubeconfig=${HOME}/.kube/config -chartsDir ${CHARTS_DIR} > log.txt
```

Comment:
What if default for `SANITY_TEST`?

Reply:
`\[Testpattern: Dynamic PV \(default fs\)\] volumes should store data`
It'll be available as a default parameter value in custom-ci pipeline.